zeroize/src/lib.rs

```diff
@@ -69,6 +69,7 @@
 //! Additionally it supports the following attribute:
 //!
 //! - `#[zeroize(drop)]`: call `zeroize()` when this item is dropped
+//! - `#[zeroize(skip)]`: skips this field or variant when calling `zeroize()`
```

Comment:
It would probably be good to note that `drop` is a type-level attribute and `skip` is a field-level attribute

Reply:
I separated them into different lists, I think it would go quiet well with #663. WDYT?